app/assets/stylesheets/components/_highlight-boxes.scss

```diff
@@ -20,11 +20,11 @@
 
 .app-c-highlight-boxes__item {
```

Comment:
It's possible that having this child element of the LI isn't necessary with flexbox. I mention this only in passing, it's not essential to investigate.